ci/test_thirdparty_nvmath.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+set -euo pipefail
+
+CUDA_VER_MAJOR_MINOR=${CUDA_VER%.*}
+
+NVMATH_PYTHON_VERSION="0.6.0"
+# The commit on Github corresponding to 0.6.0
+NVMATH_PYTHON_SHA="6bddfa71c39c07804127adeb23f5b0d2168ae38c"
```

Comment:
Non-blocking: these dependency management settings should be hoisted out into a yml file located at the project root or something similar.